### PR TITLE
Add frontend build command before "Starting The Things Stack"

### PR DESCRIPTION
**Pull request by nejraselimovic**
#### Summary
Related to #1538 that is closed but still not fixed in the documentation. 

#### Changes
The command for building the frontend is added above the command for running TTS.

#### Testing
I followed existing DEVELOPMENT.md and running TTS wouldn't work if the frontend wasn't built before.

##### Regressions
...

#### Notes for Reviewers
...

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md`. The target branch is set to `master` if the changes are fully compatible with existing API, database, configuration and CLI.
- [x] Documentation: Relevant documentation is added or updated.
- [ ] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [ ] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

**Comment by johanstokking** [comment]
Please rebase on `master` when you target that branch

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -68,6 +68,10 @@ Redis is an in-memory data store that we use as a database for "hot" data.
 
 You can use `tools/bin/mage dev:dbRedisCli` to enter a Redis-CLI shell.
 
+## Building the frontend
```
Comment (johanstokking) [inline DEVELOPMENT.md:71]: nit; Title Case

**Comment by johanstokking** [comment]
> I think it's okay to point this to `develop`, since it uses the new location of Mage (`tools/bin/mage`) which is only in develop. I don't think it's necessary to submit a separate PR for `master`.

That's fine, but there were `develop` commits in this PR targeting `master`. I'm fine with targeting `develop`

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -68,6 +68,10 @@ Redis is an in-memory data store that we use as a database for "hot" data.
...
+
+You can use `tools/bin/mage js:build` to build the frontend.
```
Comment (benolayinka) [inline DEVELOPMENT.md:73]: does `./mage` not work in `develop`? The rest of the documentation uses `./mage`

Comment (htdvisser) [reply DEVELOPMENT.md:73]: https://github.com/TheThingsNetwork/lorawan-stack/pull/2568

**Comment by johanstokking** [comment]
@nejraselimovic wat is the status here?

**Comment by nejraselimovic** [comment]
> @nejraselimovic wat is the status here?

Sorry, focused on other PR, is this ok to merge?

**Comment by johanstokking** [comment]
> Sorry, focused on other PR, is this ok to merge?

Yes, don't forget to squash the commits

**Pull request merged**